app/src/main/java/fr/free/nrw/commons/Media.java

```diff
@@ -53,6 +53,7 @@ public Media createFromParcel(Parcel parcel) {
      * Also now captions replace the previous convention of using title for filename
      */
     private String caption;
+    private String depiction;
```

Comment:
There can be zero or one or several depictions. For instance this picture shows a bridge AND an office building: https://commons.wikimedia.org/wiki/File:Momoyo-brug_in_Shibaura,_gezien_richting_het_zuidoosten,_-15_maart_2019.jpg

I suggest `List<String>`.